.gitignore

```diff
@@ -48,6 +48,7 @@ api/.env
 # Terraform
 # Local .terraform directories
 **/.terraform/*
+**/.terraform.lock.hcl
```

Comment:
Does this change allow a lock to prevent other action runs from disrupting things?

Reply:
No, this was to tell .gitignore to ignore this file. While working on this, I noticed that git was tracking this file and this file was created when I initialize terraform locally, so ideally, I don't want to commit it.